Index: target/classes/Database/tables.sql
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/target/classes/Database/tables.sql b/target/classes/Database/tables.sql
--- a/target/classes/Database/tables.sql	(revision 138e8fc1774087945d188211e8e5011fd006cd9c)
+++ b/target/classes/Database/tables.sql	(date 1766217334477)
@@ -1,0 +1,6 @@
+
+DROP DATABASE IPPSystem;
+
+CREATE DATABASE IPPSystem;
+
+CREATE TABLE 
Index: target/classes/View/hello-view.fxml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\n\n<?import javafx.geometry.Insets?>\n<?import javafx.scene.control.Label?>\n<?import javafx.scene.layout.VBox?>\n\n<?import javafx.scene.control.Button?>\n<VBox alignment=\"CENTER\" spacing=\"20.0\" xmlns:fx=\"http://javafx.com/fxml\"\n      fx:controller=\"projectgroup1.Controllers.HelloController\">\n    <padding>\n        <Insets bottom=\"20.0\" left=\"20.0\" right=\"20.0\" top=\"20.0\"/>\n    </padding>\n\n    <Label fx:id=\"welcomeText\"/>\n    <Button text=\"Hello!\" onAction=\"#onHelloButtonClick\"/>\n</VBox>\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/target/classes/View/hello-view.fxml b/target/classes/View/hello-view.fxml
--- a/target/classes/View/hello-view.fxml	(revision 138e8fc1774087945d188211e8e5011fd006cd9c)
+++ b/target/classes/View/hello-view.fxml	(date 1766217340824)
@@ -6,7 +6,7 @@
 
 <?import javafx.scene.control.Button?>
 <VBox alignment="CENTER" spacing="20.0" xmlns:fx="http://javafx.com/fxml"
-      fx:controller="projectgroup1.Controllers.HelloController">
+      fx:controller="IPPSystem.Controllers.HelloController">
     <padding>
         <Insets bottom="20.0" left="20.0" right="20.0" top="20.0"/>
     </padding>
